### PR TITLE
Restart zmq_adapter for USB0 when it dies

### PR DESCRIPTION
**Pull request by lbara**

### CODE REVIEW

File: package/zmq_adapter/src/zmq_adapter.c [hunk 1]
```diff
@@ -588,7 +592,7 @@ static ssize_t handle_write_all(handle_t *handle,
     ssize_t write_count = handle_write(handle,
                                        &((uint8_t *)buffer)[buffer_index],
                                        count - buffer_index);
-    debug_printf("wrote %zd bytes\n", write_count);
+    // debug_printf("wrote %zd bytes\n", write_count);
```
Comment (gsmcmullin) [inline package/zmq_adapter/src/zmq_adapter.c]: If we really down want this anymore, just remove it.

File: package/zmq_adapter/src/zmq_adapter.c [hunk 2]
```diff
@@ -703,8 +707,10 @@ static void io_loop_pubsub(handle_t *read_handle, handle_t *write_handle)
     /* Read from read_handle */
     uint8_t buffer[READ_BUFFER_SIZE];
     ssize_t read_count = handle_read(read_handle, buffer, sizeof(buffer));
-    debug_printf("read %zd bytes\n", read_count);
+    // debug_printf("read %zd bytes\n", read_count);
```
Comment (gsmcmullin) [inline package/zmq_adapter/src/zmq_adapter.c]: As above

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.0-branch-214-ga77e793
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-214-ga77e793/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.0-branch-214-ga77e793/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-214-ga77e793

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.0-branch-216-g0b098a5
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-216-g0b098a5/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.0-branch-216-g0b098a5/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-216-g0b098a5

**Pull request merged**